src/test/java/hill/weatherconfidence/ingest/GeocodeClientTest.java

```diff
@@ -0,0 +1,85 @@
+package hill.weatherconfidence.ingest;
+
+
+import hill.weatherconfidence.ingest.model.GeocodeItem;
+import hill.weatherconfidence.ingest.model.GeocodeResponse;
+import okhttp3.mockwebserver.MockResponse;
+import okhttp3.mockwebserver.MockWebServer;
+import okhttp3.mockwebserver.RecordedRequest;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import reactor.core.publisher.Mono;
+import reactor.test.StepVerifier;
+
+import java.io.IOException;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class GeocodeClientTest {
+
+    public static MockWebServer mockBackEnd;
+    private GeocodeClient client = null;
+
+    @BeforeAll
```

Comment:
These methods handle test lifecycle setup and destroy. Static methods run as the test is set up